main.py

```diff
@@ -13,9 +14,35 @@
                                              initial_distribution=v,
                                              num_stages=N, stopping_time=tau).create()
 
-tree.bulls_eye_plot(dot_size=5, radius=300)
+# tree.bulls_eye_plot(dot_size=5, radius=300)
+#
+# tree.set_data_at_node(4, {"a": 1})
+# print(sum(tree.probability_of_node(tree.nodes_at_stage(8))))
+#
+# print(tree)
 
-tree.set_data_at_node(4, {"a": 1})
-print(sum(tree.probability_of_node(tree.nodes_at_stage(8))))
+# RAOCP generation -----------------------------------------------------------------------------------------------------
 
-print(tree)
+Aw1 = np.eye(2)
+Aw2 = 2*np.eye(2)
+Aw3 = 3*np.eye(2)
+As = [Aw1, Aw2, Aw3]  # n x n matrices
+
+Bw1 = np.eye(2)
+Bw2 = 2*np.eye(2)
+Bw3 = 3*np.eye(2)
+Bs = [Bw1, Bw2, Bw3]  # n x u matrices
+
+cost_type = "quadratic"
+Q = 10*np.eye(2)  # n x n matrix
+R = np.eye(2)  # u x u matrix OR scalar
+Pf = 5*np.eye(2)  # n x n matrix
+
+(risk_type, alpha) = ("AVaR", 0.5)
+problem = r.core.MarkovChainRAOCPProblemBuilder(scenario_tree=tree)\
+    .with_possible_As_and_Bs(As, Bs)\
+    .with_all_cost(cost_type, Q, R, Pf)\
```

Comment:
Why can't the user specify separately the stage and terminal cost functions? You have defined a `Quadratic` function, but it is not used anywhere